### DISCUSSION TITLE
[Meta] Alignment between HUGO Nomenclature Standards Committee and Fusions Specification

### DISCUSSION THREAD

**Discussion by ahwagner** [HGVSnomenclature/hgvs-nomenclature#7] [HGVS nomenclature proposals]
The Variant Interpretation for Cancer Consortium has developed a draft, cross-consortia specification for the definition and representation of Gene Fusions. The online draft specification is available at [fusions.cancervariants.org](https://fusions.cancervariants.org/).

This discussion is an meta-thread describing the various efforts underway to align the VICC fusion specification with the [HUGO Nomenclature Standards Committees](https://www.hugo-international.org/standards/) (alignment between the HVNC, HGNC, and ISCN committees). This discussion will be updated with additional topics and status as needed. Discussion on the scope (i.e. relevant HUGO standards committees) and topics for gene fusion alignment with HUGO will be organized in this thread.

Associated discussions relevant to HVNC include:
- #10 
- #13 
- #14 
- #15 

Additional discussions peripheral to the HGVS nomenclature / HVNC:
- #16 
- #11

**Comment by reece** [comment]
@ahwagner Do you know why this discussion is locked? And what does it mean for the discussion to be locked viz the whole point of the Discussions section?

What outcome would close this discussion?

**Reply by ahwagner** [reply]
I had this locked as a "meta" thread to organize discussion. Item-specific discussion is intended to happen on (unlocked) linked threads. I don't know of a better way to do that.

Discussion should close once I update https://github.com/HGVSnomenclature/hgvs-nomenclature/pull/22 (which addresses the requirements from the above discussion threads) and it is merged into `main`.
